src/Symfony/Component/Serializer/composer.json

```diff
@@ -23,16 +23,18 @@
         "symfony/config": "~2.8|~3.0",
         "symfony/property-access": "~2.8|~3.0",
         "symfony/http-foundation": "~2.8|~3.0",
+        "symfony/cache": "~3.1",
         "doctrine/annotations": "~1.0",
         "doctrine/cache": "~1.0"
     },
     "suggest": {
-        "doctrine/annotations": "For using the annotation mapping. You will also need doctrine/cache.",
-        "doctrine/cache": "For using the default cached annotation reader and metadata cache.",
+        "symfony/cache": "For using the metadata cache.",
```

Comment:
On the validator PR, you added `"psr/cache-implementation": "For using the metadata cache.",` instead. Not that it matters so much, but for consistency, we should do the same everywhere.